CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 All notable changes to this project will be documented in this file.
 
+## ## [v0.1.10] Unreleased
+- Added: Added "api_gateway_timeout_milliseconds" variable to control the API Gateway
```

Comment:
Minor, but probably no need for the "Added: Added" - can use "Added: "
Probably have not been consistent in previous entries we have added